CHANGELOG.md

```
@@ -2,44 +1,31 @@
+## Features
 
 * **deps:** bump the all-deps group across 1 directory with 5 updates ([#189](https://github.com/VilnaCRM-Org/user-service/issues/189)) ([c800529](https://github.com/VilnaCRM-Org/user-service/commit/c80052918cfbd1404ee77b0ef84d1dc4a3c96b97))
 
+## [2.5.0](https://github.com/VilnaCRM-Org/user-service/compare/v2.4.1...v2.5.0) (2025-04-14)
 
-
-# [2.5.0](https://github.com/VilnaCRM-Org/user-service/compare/v2.4.1...v2.5.0) (2025-04-14)
-
-
-### Features
+## Features
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)